File: java-spring-rest-mvc/src/main/java/com/spring_guru/spring_rest_mvc/models/Beer.java
package com.spring_guru.spring_rest_mvc.models;

import lombok.Data;

import java.math.BigDecimal;
import java.time.LocalDateTime;
import java.util.UUID;

@Data
public class Beer {
    private UUID id;
    private Integer version;
    private Integer quantity;
    private String name;
    private String upc;
    private BeerStyle style;
    private BigDecimal price;
    private LocalDateTime createDate;
    private LocalDateTime updateDate;
}

File: kotlin-spring-rest-mvc/src/main/kotlin/com/kotlin_spring_rest_mvc/kotlin_spring_rest_mvc/models/Beer.kt
package com.kotlin_spring_rest_mvc.kotlin_spring_rest_mvc.models

import java.math.BigDecimal
import java.time.LocalDateTime
import java.util.*

data class Beer(
    var id: UUID? = null,
    var version: Int? = null,
    var quantity: Int? = null,
    var name: String? = null,
    var upc: String? = null,
    var style: BeerStyle? = null,
    var price: BigDecimal? = null,
    var createDate: LocalDateTime = LocalDateTime.now(),
    var updateDate: LocalDateTime = LocalDateTime.now()
)